test/e2e/pipeline/pipeline_test.go

```diff
@@ -305,6 +305,23 @@ Waiting for logs to be available...
 		}
 	})
 
+	t.Run("Start PipelineRun with --taskrun-spec", func(t *testing.T) {
+		tkn.MustSucceed(t, "pipeline", "start", tePipelineName,
+			"-p=filename=output",
+			"-w=name=shared-data,emptyDir=",
+			"--taskrun-spec="+helper.GetResourcePath("/taskrunspec/taskrunspec.yaml"),
```

Comment:
the path seems incorrect here

Reply:
fixed @ https://github.com/tektoncd/cli/pull/1969/commits/ede20795c9f389f988878649b005797f9941f0a4